dagger-common/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
`Descriptors.FieldDescriptor` is a final class. In order to mock it, we need to enable this mockito extension

Ref: https://www.baeldung.com/mockito-final#configure-mocktio

Reply:
Using powermockito instead